ddtrace/__init__.py

```diff
@@ -7,13 +7,7 @@
 
 ModuleWatchdog.install()
 
-# Acquire a reference to the threading module. Some parts of the library (e.g.
-# the profiler) might be enabled programmatically and therefore might end up
-# getting a reference to the tracee's threading module. By storing a reference
-# to the threading module used by ddtrace here, we make it easy for those parts
-# to get a reference to the right threading module.
-import threading as _threading
-
+import ddtrace.internal._unpatched  # noqa
```

Comment:
Perhaps we should add a comment here saying that this import acquires the references to the unpatched modules.

Reply:
```suggestion
# Ensure we capture references to unpatched modules as early as possible
import ddtrace.internal._unpatched  # noqa
```